src/python/system/environment.py

```diff
@@ -319,8 +324,12 @@ def get_environment_settings_as_string():
       if not environment_variable_value:
         continue
 
-      environment_string += '[Environment] %s = %s\n' % (
-          environment_variable, environment_variable_value)
+      if platform() == 'WINDOWS':
+        environment_string += '[Environment] set "%s=%s"\n' % (
```

Comment:
i don't think quotes around %s=%s work, it should %s="%s"

Reply:
Shouldn't matter after the change to remove export/set.